pom.xml

```
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>4.2</version>
+    <relativePath/>
+  </parent>
+  <groupId>io.jenkins.plugins</groupId>
+  <artifactId>dark-theme</artifactId>
+  <version>${revision}${changelist}</version>
+  <packaging>hpi</packaging>
+  <properties>
+    <revision>0.1</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jenkins.version>2.240</jenkins.version>
+    <java.level>8</java.level>
+    <tagNameFormat>@{project.version}</tagNameFormat>
+  </properties>
+  <name>Dark theme</name>
```

move the name and description up?

```suggestion
  <name>Dark Theme</name>
```

---

This came from the archetype I think, not sure if worth moving